### PR TITLE
fix: fix missmatched request paths, updated docs

### LINKED ISSUE #24: [CAM] Update docs for new IsoStretcherProcessImg
@DanielSeehausen Might actually makes more sense to just link directly to https://app.swaggerhub.com/apis/OSPIN-Webapp/ospin-rest_api/dev#/Processes/post_process__processId__functionalities__functionalityId_

to keep a single source of truth (I added request body definitions to the hambda pr)

### LINKED ISSUE #25: Fix Path for createFromImage and for CreatefromURI
A rogue backslash at the beginning of the api
```
/processes/${processId}/functionalities/${functionalityId}/images,
should be
processes/${processId}/functionalities/${functionalityId}/images,
```

### PR DESCRIPTION
**Pull request by FelixMZ2018**

### CODE REVIEW

File: src/deviceAPI/process/functionality/image/createFromURI.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const AuthorizedDeviceAPI = require('../../../AuthorizedDeviceAPI')
 
 module.exports = serializeAxiosResponse(
   (processId, functionalityId, body) => AuthorizedDeviceAPI.post(
-    `/processes/${processId}/functionalities/${functionalityId}/images`,
-    { body, ...DEFAULT_REQ_OPTS },
+    `processes/${processId}/functionalities/${functionalityId}/images`,
+    { ...body }, { ...DEFAULT_REQ_OPTS },
```
Comment (vdiezel) [inline src/deviceAPI/process/functionality/image/createFromURI.js]: This syntax seems wierd: 

`{ ...body }` is the same as jsut passing `body` (expect that you are creating a shallow clone, but I don't think this is the intent here); same applies for `DEFAULT_REQ_OPTS`

File: test/src/deviceAPI/process/functionality/image/createFromFile.test.js [hunk 2]
```diff
@@ -28,12 +28,12 @@ describe('create Process Functionality Image', () => {
...
     expect(AuthenticatedDeviceAPI.post).toHaveBeenCalledWith(
-      `/processes/${processId}/functionalities/${functionalityId}/images`,
-      { body: {
+      `processes/${processId}/functionalities/${functionalityId}/images`,
+      {
         imageCreatedAt,
         imageDataUri: expect.anything(),
       },
-      ...DEFAULT_REQ_OPTS },
+      { ...DEFAULT_REQ_OPTS },
```
Comment (vdiezel) [inline test/src/deviceAPI/process/functionality/image/createFromFile.test.js]: see comment above

File: test/src/deviceAPI/process/functionality/image/createFromURI.test.js [hunk 3]
```diff
@@ -27,8 +27,8 @@ describe('create Process Functionality Image', () => {
     )
 
     expect(AuthenticatedDeviceAPI.post).toHaveBeenCalledWith(
-      `/processes/${processId}/functionalities/${functionalityId}/images`,
-      { body, ...DEFAULT_REQ_OPTS },
+      `processes/${processId}/functionalities/${functionalityId}/images`,
+      body, { ...DEFAULT_REQ_OPTS },
```
Comment (vdiezel) [inline test/src/deviceAPI/process/functionality/image/createFromURI.test.js]: see comment above

**Review by vdiezel — DISMISSED** [review]
Looks good - my only complaint is the redundant destructuring of object into new objects!

File: src/deviceAPI/process/functionality/image/createFromURI.js (hunk 1 above)
Comment (FelixMZ2018) [reply src/deviceAPI/process/functionality/image/createFromURI.js]: Good catch! I tink the opts part was meant for passing in additional opts which we dont use so I just changed it for it now, also taken out the destructuring from the body and tested it and it all seems to be going through 

Thanks!

**Review by vdiezel — APPROVED** [review]

**Pull request merged**